soc/espressif/esp32s3/soc.c

```diff
@@ -211,6 +185,13 @@ void IRAM_ATTR __esp_platform_start(void)
 
 	esp_intr_initialize();
 
+#if CONFIG_ESP_SPIRAM
+	/* Init Shared Multi Heap for PSRAM */
+	if (esp_psram_smh_init()) {
+		abort();
```

Comment:
Question, is abort() the best approach here? PSRAM that fails in its initialization does not mean the entire system has been crashed, so a LOG_ERR instead would offer a more descriptive diagnostics, and avoid a (initially at least) cycle of rebooting when something in the PSRAM fails.